pkg/reconciler/kubernetes/tektonchain/tektonchain.go

```diff
@@ -230,8 +237,10 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonChain
 			if tc.Spec.GetTargetNamespace() == pipelineNamespace {
 				manifest = manifest.Filter(mf.Not(mf.ByKind("Namespace")))
 			}
-			// remove secret from this installerset as this installerset will be deleted on upgrade
-			manifest = manifest.Filter(mf.Not(mf.ByKind("Secret")))
+			// remove secret and `chains-config` configMap from this installerset as this installerset will be deleted on upgrade
```

Comment:
this shoule be - `remove secret from this installerset as this installerset will be deleted on upgrade and `chains-config` configMap  as it will be in separate installerset`